{{cookiecutter.project_slug}}/README.md

```diff
@@ -79,3 +103,13 @@ This project is set up to run using [Docker Compose](https://docs.docker.com/com
 2. Run `python manage.py makemigrations`
```

Comment:
what about having a table here like:

| Command | Description |
| --- | --- |
| `python manage.py makemigrations` | Make migrations |

I know they are pretty obviuos but it may help

Reply:
That's a good suggestion as well but I feel like if we do that for PIPENV setup then we should also do it for the DOCKER setup. These instructions are only meant to be a way to quickstart the local dev environment.

Reply:
We can have three columns, Description, PIPENV, DOCKER. That might make more sense.

Reply:
You are right. I guess that we can have another part for commands that devs can run inside the shell ( for testing, for making migration files, etc ) in order not to have differences between the Docker setup ( that I believe its correct because it has steps to follow ) and the Pipenv setup.